packages/node/__tests__/index.test.ts

```diff
@@ -106,13 +70,51 @@ describe('#metrics', () => {
 
   afterEach(() => {
     nock.cleanAll();
+  });
+
+  it('should throw an error if `apiKey` is missing', () => {
+    const app = express();
+    app.use((req, res, next) => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      readmeio.log('', req, res, {});
+      return next();
+    });
+    app.get('/test', (req, res) => res.sendStatus(200));
+
+    // This silences console.errors from express internals
+    app.set('env', 'test');
```

Comment:
What stuff is Express logging that we're silencing here?

Reply:
c311e013acd02973c72ad4302c81c980cb9cc41f